release/gcb/gcb_lib.sh

```diff
@@ -142,3 +143,94 @@ function make_istio() {
     "${EXTRA_ARTIFACTS:-$PWD/LICENSES.txt}"
 }
 
+fix_values_yaml() {
+  local VERSION="$1"
+  local DOCKER_HUB="$2"
+  update_helm "istio-${VERSION}-linux.tar.gz" "${VERSION}" "${DOCKER_HUB}"
+  update_helm "istio-${VERSION}-osx.tar.gz" "${VERSION}" "${DOCKER_HUB}"
+  update_helm "istio-${VERSION}-win.zip" "${VERSION}" "${DOCKER_HUB}"
+}
+
+function update_helm() {
+  if [ -z "${LOCAL_BUILD+x}" ]; then
+    gsutil -q cp "gs://${CB_GCS_BUILD_PATH}/${tarball_name}" .
+  fi
+  local tarball_name="$1"
+  local VERSION="$2"
+  local DOCKER_HUB="$3"
+  if [[ ${tarball_name} == *.zip ]]; then
+    local unzip_cmd="unzip -q"
+    local zip_cmd="zip -q -r"
+  else
+    local unzip_cmd="tar -zxf"
+    local zip_cmd="tar -zcf"
```

Comment:
maybe not in this pr, but shouldn't we do this before tarring and building mac/linux/windows builds?

Reply:
Will look into it.